### DISCUSSION TITLE
Conversation Starters for Assistants

### DISCUSSION THREAD

**Discussion by secondtruth** [mckaywrigley/chatbot-ui#1442] [Ideas]
I propose adding a "Conversation Starters" feature for assistants, like in ChatGPT. Before starting a conversation, this feature would offer users prompts defined by the assistant creator to initiate engaging discussions or trigger frequent actions.

## Potential Future Enhancements
- Implement answer suggestions for running conversations, similar to Microsoft Copilot (formerly Bing Chat), to provide users with quick replies or actions based on the conversation context.
- Integration with external APIs to pull in real-time data or trending topics as conversation starters.

## See also
- https://github.com/mckaywrigley/chatbot-ui/discussions/1452
- https://github.com/mckaywrigley/chatbot-ui/discussions/1336

**Comment by masonjames** [comment]
Agree! I think wherever possible it would be nice to have definable defaults - conversation starters would be a great one.
